dartsim/src/JointFeatures.cc

```diff
@@ -808,6 +808,35 @@ Wrench3d JointFeatures::GetJointTransmittedWrenchInJointFrame(
   wrenchOut.force = transmittedWrenchInJoint.tail<3>();
   return wrenchOut;
 }
+
+/////////////////////////////////////////////////
+//! @note This setting applies to all DynamicJointConstaints
+void JointFeatures::SetConstraintForceMixing(const Identity &_childID,
+    double _value)
+{
+  auto linkInfo = this->ReferenceInterface<LinkInfo>(_childID);
```

Comment:
if we pass a joint ID instead of the child link ID, I think we could extract the child LinkInfo using the [same approach as in `DetachJoint`](https://github.com/gazebosim/gz-physics/blob/f1643e4a2349f1e6ae1a61c2e38d53cef2a4f30a/dartsim/src/JointFeatures.cc#L458-L468)